### PR TITLE
Updated log format

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: AEPMedia/Sources/MediaCollection/MediaOfflineSession.swift [hunk 1]
```diff
@@ -15,9 +15,10 @@ import AEPServices
 
 class MediaOfflineSession: MediaSession {
 
-    private let LOG_TAG = "MediaOfflineSession"
+    private static let LOG_TAG = MediaConstants.LOG_TAG
+    private static let CLASS_NAME = "MediaOfflineSession"
     private static let MAX_ALLOWED_FAILURE = 3 //The maximun number of times SDK retries to send hit on failure, after that drop the hit.
-    private static let DURATION_BETWEEN_HITS_ON_FAILURE = 60 // Retry duration in case of failure
+    private static let DURATION_BETWEEN_HITS_ON_FAILURE = 10//60 // Retry duration in case of failure
```
Comment (cacheung) [inline AEPMedia/Sources/MediaCollection/MediaOfflineSession.swift:21]: change back to 60 before merge

**Review by praveek — APPROVED** [review]

File: AEPMedia/Sources/MediaTracker.swift [hunk 2]
```diff
@@ -14,30 +14,57 @@ import Foundation
...
 
+    /// API to track an error in media playback.
+    /// - Parameter:
+    ///   - errorId: `String` Identifier describing the error.
     @objc(trackError:)
     func trackError(errorId: String)
 
+    /// API to track media event.
+    /// - Parameter:
+    ///   - event: `MediaEvent` describing the event to track `ChapterStart`, `ChapterComplete`, `AdBreakStart`, `AdBreakComplete`, `AdStart`, `AdComplete`, `SeekStart`, `SeekComplete`, `BufferStart`,
```
Comment (cacheung) [inline AEPMedia/Sources/MediaTracker.swift:49]: Missing StateStart and StateEnd

File: AEPMedia/Sources/MediaTracker.swift [hunk 3]
```diff
@@ -14,30 +14,57 @@ import Foundation
...
+    ///   `BufferComplete`, `BitrateChange`.
+    ///   - info: `Dictionary` created using `createChapterObject`, `createAdBreakObject`, `createAdObject`, `createStateObject` API  with for `AdBreakStart`, `AdStart`, `ChapterStart`, `StateStart` and `StateEnd` events respectively. Pass nil for other events.
```
Comment (cacheung) [inline AEPMedia/Sources/MediaTracker.swift:51]: Matching the order
- info: `Dictionary` created using `createChapterObject`, `createAdBreakObject`, `createAdObject`, `createStateObject` API with for `ChapterStart`, `AdBreakStart`, `AdStart`, `StateStart` and `StateEnd` events respectively.

File: AEPMedia/Tests/UnitTests/Media+PublicAPITests.swift [hunk 4]
```diff
@@ -155,19 +155,19 @@ class MediaPublicAPITests: XCTestCase {
...
         // < 1 position
-        infoMap = Media.createAdObjectWith(name: "adName", adId: "AdId", position: 0, length: 20)
+        infoMap = Media.createAdObjectWith(name: "adName", id: "AdId", position: 0, length: 20)
         XCTAssertNil(infoMap)
 
         // < 0 length
-        infoMap = Media.createAdObjectWith(name: "adName", adId: "AdId", position: 2, length: -1)
+        infoMap = Media.createAdObjectWith(name: "adName", id: "AdId", position: 2, length: -1)
         XCTAssertNil(infoMap)
     }
```
Comment (cacheung) [inline AEPMedia/Tests/UnitTests/Media+PublicAPITests.swift:173]: Can you also change for SampleMediaApp?
https://github.com/adobe/aepsdk-media-ios/blob/dev/Samples/MediaSampleApp/Analytics/VideoAnalyticsProvider.swift#L166

File: AEPMedia/Sources/MediaCollection/MediaSession.swift [hunk 5]
```diff
@@ -73,22 +74,22 @@ class MediaSession {
 
     /// Notifies MediaState updates
     func handleMediaStateUpdate() {
-        Log.warning(label: LOG_TAG, "\(#function) - This function should be handle by the child class.")
+        Log.warning(label: Self.LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - This function should be handle by the child class.")
```
Comment (praveek) [inline AEPMedia/Sources/MediaCollection/MediaSession.swift:77]: should be handled

File: AEPMedia/Sources/MediaCollection/MediaSession.swift [hunk 6]
```diff
@@ -73,22 +74,22 @@ class MediaSession {
...
     }
 
     ///Includes the business logic for ending session. Implemented by more concrete classes of MediaSession: `MedialRealTimeSession` and `MediaOfflineSession`.
     func handleSessionEnd() {
-        Log.warning(label: LOG_TAG, "\(#function) - This function should be handle by the child class.")
+        Log.warning(label: Self.LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - This function should be handle by the child class.")
```
Comment (praveek) [inline AEPMedia/Sources/MediaCollection/MediaSession.swift:82]: should be handled

File: AEPMedia/Sources/MediaCollection/MediaSession.swift [hunk 7]
```diff
@@ -73,22 +74,22 @@ class MediaSession {
...
     }
 
     ///Includes the business logic for aborting session. Implemented by more concrete classes of MediaSession: `MedialRealTimeSession` and `MediaOfflineSession`.
     func handleSessionAbort() {
-        Log.warning(label: LOG_TAG, "\(#function) - This function should be handle by the child class.")
+        Log.warning(label: Self.LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - This function should be handle by the child class.")
```
Comment (praveek) [inline AEPMedia/Sources/MediaCollection/MediaSession.swift:87]: should be handled

File: AEPMedia/Sources/MediaCollection/MediaSession.swift [hunk 8]
```diff
@@ -73,22 +74,22 @@ class MediaSession {
...
     }
 
     ///Includes the business logic for queuing `MediaHit`. Implemented by more concrete classes of MediaSession: `MedialRealTimeSession` and `MediaOfflineSession`.
     /// - Parameter hit: `MediaHit` to be queued.
     func handleQueueMediaHit(hit: MediaHit) {
-        Log.warning(label: LOG_TAG, "\(#function) - This function should be handle by the child class.")
+        Log.warning(label: Self.LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - This function should be handle by the child class.")
```
Comment (praveek) [inline AEPMedia/Sources/MediaCollection/MediaSession.swift:93]: should be handled

**Review by praveek — APPROVED** [review]

**Review by praveek — APPROVED** [review]

**Pull request merged**